docs/backup-restore.md

```diff
@@ -1,134 +1,204 @@
 
-The ways to create transactional backups are different among each database product.
-We show examples for some database products, but you are requested to find safe ways for a transactional backup from your databases at your own risk.
+{% capture notice--warning %}
+**Attention**
 
-#### MySQL (backup)
+Before creating a backup, you should consider the safest way to create a transactionally consistent backup of your databases and understand any risks that are associated with the backup process.
+{% endcapture %}
 
-Use the `mysqldump` command with `--single-transaction` option.
+<div class="notice--warning">{{ notice--warning | markdownify }}</div>
 
-#### PostgreSQL (backup)
+<div id="tabset-1">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'Amazon_RDS_or_Azure_Database_for_MySQL_or_PostgreSQL1', 'tabset-1')" id="defaultOpen-1">Amazon RDS or Azure Database for MySQL or PostgreSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'MySQL1', 'tabset-1')">MySQL</button>
+  <button class="tablinks" onclick="openTab(event, 'PostgreSQL1', 'tabset-1')">PostgreSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'SQLite1', 'tabset-1')">SQLite</button>
+</div>
+
+<div id="Amazon_RDS_or_Azure_Database_for_MySQL_or_PostgreSQL1" class="tabcontent" markdown="1">
+
+You can restore to any point within the backup retention period by using the automated backup feature.
+</div>
+<div id="MySQL1" class="tabcontent" markdown="1">
+
+Use the `mysqldump` command with the `--single-transaction` option.
+</div>
+<div id="PostgreSQL1" class="tabcontent" markdown="1">
 
 Use the `pg_dump` command.
+</div>
+<div id="SQLite1" class="tabcontent" markdown="1">
+
+Use the `.backup` command with the `.timeout` command as specified in [Special commands to sqlite3 (dot-commands)](https://www.sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_)
+
+For an example, see [BASH: SQLite3 .backup command](https://stackoverflow.com/questions/23164445/bash-sqlite3-backup-command).
+</div>
+</div>
+
+### Back up with explicit pausing
+
+Another way to create a transactionally consistent backup is to create a backup while ScalarDB Cluster does not have any outstanding transactions. Creating the backup depends on the following:
+
+- If the underlying database has a point-in-time snapshot or backup feature, you can take a snapshot during the period.
+- If the underlying database has a point-in-time restore or recovery (PITR) feature, you can set a restore point to a time (preferably the mid-time) in the pause duration period.
+
+To make ScalarDB drain outstanding requests and stop accepting new requests so that a pause duration can be created, you should use [ScalarDB Server](scalardb-server.md), which uses the `scalar-admin` interface, or implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application that uses ScalarDB.
```

Comment:
Should we change `ScalarDB Server` to `ScalarDB Cluster` here?

 ```suggestion
To make ScalarDB drain outstanding requests and stop accepting new requests so that a pause duration can be created, you should use ScalarDB Cluster, which uses the `scalar-admin` interface, or implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application that uses ScalarDB.
```